readme.md

```diff
@@ -77,6 +78,8 @@ $ capture-website --help
     --user-agent="I love unicorns"
     --cookie="id=unicorn; Expires=Wed, 21 Oct 2018 07:28:00 GMT;"
     --authentication="username:password"
+		--inset=100
+	  --element=".main-content" --inset=-10
```

Comment:
Incorrect indentation

Reply:
my bad, it was looking good in vs code, fixed.